int64[] destination_id
float64[] distance
int64[] rssi
time[] stamp
